src/mixtures/mixturemodel.jl

```diff
@@ -254,7 +254,7 @@ end
 
 #### show
 
-function show(io::IO, d::MixtureModel)
+function show(io::IO, d::AbstractMixtureModel)
     K = ncomponents(d)
     pr = probs(d)
     println(io, "MixtureModel{$(component_type(d))}(K = $K)")
```

Comment:
Probably this should be modified since in general the distribution is not a `MixtureModel`.

Reply:
Maybe by putting space between two words.

Reply:
Not directly related to the change here but the two argument show methods shouldn't have line breaks so I think this should be changed to a three argument show methods.

Reply:
@devmotion Are you okay with the updated version?